common/OpTestSystem.py

```diff
@@ -556,6 +556,8 @@ def wait_for_it(self, **kwargs):
             # check console type and pass 5 to skip SMS menu when booting an LPAR
             if isinstance(self.console, OpTestHMC.HMCConsole) and x == 1:
                 sys_pty.sendline('5')
+                sys_pty.sendline()
+                time.sleep(60)
```

Comment:
The problem is not resolved with 60 sec time sleep

Reply:
ah this not good to just passing sleep as discussed    we should set profile in normal boot before running test if not we can handle in code

Reply:
Profile is already was in normal boot, but still it fails,, sleep is needed

Reply:
ah tricky than as i see we pass `sendline('5')` as that for boot , any way we can discuss this offline